notes.md

```diff
@@ -0,0 +1,123 @@
+
+## Notes on SAP Hana file lineage ingestion  
+
+### Connecting to SAP Hana
+
+1. Run the datahub-sap-hana-example database using ```docker-compose up -d```. Follow instructions [here].(https://github.com/contiamo/datahub-sap-hana-example)
```

Comment:
There show be no references to our internal repo or any of the configuration or data in this repo. Either move notes there or make this slightly more generic by linking to the SAP docs for testing SAP in docker